src/Demos/2-distributing-work.fsx

```diff
@@ -19,24 +19,12 @@ open Nessos.Streams
 // First connect to the cluster
 let cluster = Runtime.GetHandle(config)
 
-// create two jobs (but don't execute them)
-let workflowA = cloud { return "hello world from A" }
```

Comment:
I've updated this PR with a bunch of micro improvements and fixes

Here I have removed the `<||>`` operator from the tutorial sequence.  I've given five 1:1 tutorials in the last couple of days and every time I've found myseff skipping this - it just isn't a critical part of the programming model compared with Cloud.Parallel or Cloud.Choice.